truenas_installer/install.py

```diff
@@ -0,0 +1,164 @@
+import tempfile
+
+import json
+
+import asyncio
```

Comment:
We should fix the imports here. Alphabetical order and no reason to include spaces between the builtin libraries.